documentation/api/query/v4/entities.markdown

```diff
@@ -0,0 +1,53 @@
+---
+title: "PuppetDB 3.2 » API » v4 » Entities"
+layout: default
+canonical: "/puppetdb/latest/api/query/v4/entities.html"
+---
+
+[aggregate-event-counts]: ./aggregate-event-counts.html
+[catalogs]: ./catalogs.html
+[edges]: ./edges.html
+[environments]: ./environments.html
+[event-counts]: ./event-counts.html
+[events]: ./events.html
+[fact-names]: ./fact-names.html
+[facts]: ./facts.html
+[fact-contents]: ./fact-contents.html
+[fact-paths]: ./fact-paths.html
+[nodes]: ./nodes.html
+[query]: ./query.html
+[reports]: ./reports.html
+[resources]: ./resources.html
+[root]: ./index.html
+[from]: ./operators.html#context-operators
+[subquery]: ./operators.html#subquery-operators
+
+The PuppetDB API provides access to a series of data entities that map to the Puppet eco-system
+and the data that PuppetDB stores.
+
+## Entity Types
+
+The following table shows the list of entities available, and their respective REST endpoints for
+direct querying.
+
+The entity name is utilised within queries themselves, in particular within the [`from`][from]
+and [`subquery`][subquery] operators.
```

Comment:
Yeah, its basically replacing select_<entity> ... and providing upper and lower-level syntax consistency in doing so. My decision to move the 'from' part to the outside though, is more that even in things like SQL, the table context is the mandatory part, extract is not mandatory in principle (ie. select * from foo).